### PR TITLE
Add contributors to AUTHORS.md

### PR DESCRIPTION
**Pull request by santisoler**
Add contributors to Harmonica that where missing on the AUTHORS.md file.

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If adding new functionality, add an example to the docstring, gallery, and/or tutorials.

### CODE REVIEW

**Comment by santisoler** [comment]
Hi @birocoles, @ziebam, @nshea3, @aguspesce.

We are planning to make a new release for Harmonica including all the features we have developed so far. We consider some pieces of Harmonica are ready to be used and this may be a good opportunity to start doing so, not only for taking advantages of what we've already developed, but also for receiving feedback.

I took the liberty to add you all to the `AUTHORS.md` file, where all the contributors to the project should be listed. But I don't want to merge this PR without your approval. Please let me know if you want to be listed or not and if the name and url I've chosen for each one of you is correct or should be modified.

**Comment by santisoler** [comment]
By the way, if you have an ORCID profile, please paste its number here so we can link it on the Zenodo release we will make for Harmonica, similar to the one for Rockhound: https://zenodo.org/record/3627166

**Comment by ziebam** [comment]
Perfectly fine by me.

Thanks for the attribution!

**Comment by santisoler** [comment]
> Perfectly fine by me.
> 
> Thanks for the attribution!

No problem @ziebam! By the way, would you like to give your full name so we can replace your GitHub username with it?

**Comment by nshea3** [comment]
Fine with me as well. Thanks! 

My ORCID is https://orcid.org/0000-0001-6007-5686. I am currently a master's student at MINES ParisTech.

**Comment by aguspesce** [comment]
HI! 
I forgot to write my information. My ORCID is https://orcid.org/0000-0002-5538-8845

**Comment by ziebam** [comment]
@santisoler Hi and sorry for the late reply! Actually I'd like to stay _somewhat_ anonymous with my GitHub username.

**Comment by santisoler** [comment]
> @santisoler Hi and sorry for the late reply! Actually I'd like to stay _somewhat_ anonymous with my GitHub username.

Ok @ziebam. No problem. I can list you with your GitHub name on Zenodo.

Nevertheless I must say that if we decide to submit an Harmonica paper to [JOSS](https://joss.theoj.org/) in the future , all contributors that wants to appear on the authors list must give a full name.

**Comment by leouieda** [comment]
Hi @ziebam @nshea3 @aguspesce @birocoles as introduced in fatiando/contributing#13 we have new authorship guidelines for Zenodo archives and publications. We really appreciate your help with Harmonica and want to make sure everyone is credited.

The only change that is relevant here is: Authors that will be included in Zenodo archives must have their full name, affiliation, and ORCID in the `AUTHORS.md` file. This is so releasing Harmonica is as frictionless for us as possible. 

I'm assuming that @nshea3 @aguspesce @birocoles are OK with having their affiliation and ORCID in the `AUTHORS.md` file. **Let us know if you'd rather not**.

@ziebam I understand your desire to be anonymous. We'll still include you in the AUTHORS file and the changelog but won't be able to add you to the Zenodo archive. Thank you for your help with Harmonica. It's much appreciated!

**Comment by leouieda** [comment]
@nshea3 @aguspesce @birocoles @santisoler I added your information to the AUTHORS file. Please check that everything is correct before we merge.

File: AUTHORS.md [hunk 1]
```diff
@@ -1,7 +1,11 @@
...
-last name) and are considered "The Harmonica Developers":
+The following people have made contributions to the project (in alphabetical
+order by last name) and are considered "The Harmonica Developers":
 
-* [Santiago Soler](https://github.com/santisoler)
-* [Leonardo Uieda](https://github.com/leouieda)
+* [Vanderlei C Oliveira Jr](https://github.com/birocoles) - Observatório Nacional, Brazil (ORCID: 0000-0002-6338-4086)
+* [Agustina Pesce](https://github.com/aguspesce) - Universidad Nacional de San Juan, Argentina (ORCID: 0000-0002-5538-8845)
+* [Nicholas Shea](https://github.com/nshea3) - MINES ParisTech (ORCID: 0000-0001-6007-5686)
+* [Santiago Soler](https://github.com/santisoler) - Universidad Nacional de San Juan, Argentina (ORCID: 0000-0001-9202-5317)
```
Comment (santisoler) [inline AUTHORS.md]:
```suggestion
* [Santiago Soler](https://github.com/santisoler) - CONICET, Argentina; Instituto Geofísico Sismológico Volponi, Universidad Nacional de San Juan, Argentina (ORCID: 0000-0001-9202-5317)
```

**Pull request merged**